TypeApprovalRegister+services.yaml

```diff
@@ -173,10 +218,12 @@ clients:
       application-name: OamLog
       release-number: 1.0.0
       uuid: tar-1-0-0-http-c-0050
+            tar-1-0-0-http-c-bm-ol-1-0-0-000
```

Comment:
As per Structure of uuid definition. api-segment for OL http-c and tcp-c would be "bs", since we have only one op-c under EatL that too "bs"